### PR TITLE
ISSUE-9: Clean individual files when CreateSubdirectories is false

### LINKED ISSUE #9: LocalBackup - cleanup also files if not CreateSubdirectories flag is off

### PR DESCRIPTION
**Pull request by Svenskapojkarna**
Fixes #9

### CODE REVIEW

File: Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs [hunk 1]
```diff
@@ -75,11 +75,16 @@ private static List<string> CleanUp(Input input, string backupDirectory, Cancell
 
             if (!input.CreateSubdirectories)
             {
-                foreach (var dir in Directory.GetDirectories(backupDirectory).Where(f => new DirectoryInfo(f).LastWriteTime < DateTime.Now.AddDays(-input.DaysOlder)))
+                foreach (var dir in Directory.GetDirectories(backupDirectory))
```
Comment (RikuVirtanen) [inline Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs:78]: Is it possible in Linux that the GetDirectories() method returns also dirs: '.' and '..'? If so there should be a check that those are not deleted.

File: Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs [hunk 2]
```diff
@@ -75,11 +75,16 @@ private static List<string> CleanUp(Input input, string backupDirectory, Cancell
...
+                    if (Directory.GetLastWriteTime(dir) < DateTime.Now.AddDays(-input.DaysOlder)) Directory.Delete(dir, true);
                     result.Add($"{dir} deleted.");
                 }
+                foreach (var file in Directory.GetFiles(backupDirectory))
+                {
+                    if (File.GetLastWriteTime(file) < DateTime.Now.AddDays(-input.DaysOlder)) File.Delete(file);
+                    result.Add($"{file} deleted.");
+                }
             }
             else
```
Comment (RikuVirtanen) [inline Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs:89]: Same thing in the else path.

**Review by RikuVirtanen — CHANGES_REQUESTED** [review]

File: Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs (hunk 1 above)
Comment (Svenskapojkarna) [reply Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs:78]: Seems like they don't. Method only returns directories in the directory.

File: Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs (hunk 2 above)
Comment (Svenskapojkarna) [reply Frends.Files.LocalBackup/Frends.Files.LocalBackup/LocalBackup.cs:89]: Same thing here.

**Review by RikuVirtanen — APPROVED** [review]

**Pull request merged**